test/integration.js

```diff
@@ -55,15 +54,6 @@ describe('integrations', function() {
     taker.parallel('test')(done);
   });
 
-  it('should lints all piped files', function(done) {
-    taker.task('test', function() {
-      return vinyl.src('./fixtures/test.js', { cwd: __dirname })
-        .pipe(jshint());
-    });
-
-    taker.parallel('test')(done);
-  });
```

Comment:
Ah, this test was ensuring that ending without a writable stream worked. I can add that test back